zorm/src/main/kotlin/com/zenmo/orm/companysurvey/SurveyRepository.kt

```diff
@@ -267,6 +267,7 @@ class SurveyRepository(
             expansionPlans = row[GridConnectionTable.expansionPlans],
             electrificationPlans = row[GridConnectionTable.electrificationPlans],
             surveyFeedback = row[GridConnectionTable.surveyFeedback],
+            pandIds = row[GridConnectionTable.pandIds].map { PandID(it) }.toSet(),
```

Comment:
No test related?

Reply:
This is tested in the "full" save and load of mockSurvey at https://github.com/Zenmo/zero-web/blob/16d9c9dadec64222503e05df61108d0556b0fa92/zorm/src/test/kotlin/com/zenmo/orm/companysurvey/RepositoryTest.kt#L71